packages/babel-plugin/__tests__/stylex-transform-define-vars-test.js

```diff
@@ -317,6 +317,41 @@ describe('@stylexjs/babel-plugin', () => {
       `);
     });
 
+    test('transforms variables object with numeric keys and add stylex.inject in debug mode', () => {
+      expect(
+        transform(
+          `
+          import stylex from 'stylex';
+          export const buttonTheme = stylex.defineVars({
+            '10': {
+              default: 'blue',
+              '@media (prefers-color-scheme: dark)': 'lightblue',
+              '@media print': 'white',
+            },
+            '1.5 pixels': {
```

Comment:
I think we'd be better off not supporting stupid key names

Reply:
To be fair, these are not supposed to be the CSS variables themselves. These are just JS references to CSS variables whose names are auto-generated. As this PR shows, we can easily escape the names for debug names.

We allow arbitrary strings in other StyleX APIs like `stylex.create`. Adding an arbitrary limitation on what you can use in the object references while creating CSS variables doesn't seem to help and makes the API more inconsistent.

Reply:
One is a key that doesn't get turned into any CSS code, the other is not.